pyglossary/plugins/cc_cedict/summarize.py

```diff
@@ -0,0 +1,16 @@
+import re
+import string
+
+parenthetical = re.compile(r'\([^)]+?\)')
+punct_table = {ord(p): ' ' for p in string.punctuation if p not in "-'"}
```

Comment:
Mix of double quotes and single quotes in the same file. Please replace single quotes with double quotes, because the whole repo is double quote. Except for the files you did not write (took from somewhere else)